table.csv

```diff
@@ -35,6 +35,7 @@ multihashes,,
 sha1,               ,                         0x11
 sha2-256,           ,                         0x12
 sha2-512,           ,                         0x13
+dbl-sha2-256,       ,                         0x56
```

Comment:
> One could argue that a "n times hash function" is a whole new function

Do you mean a new function on multihash's API, or a new hash function? I think for multihash it's simplest to consider it a new hash function. Apply-n-times is far from the only variation in usage of a hash function, so that might become very complicated very quickly.